client/src/components/ElementEditor/Header.js

```diff
@@ -174,6 +198,9 @@ Header.propTypes = {
   previewExpanded: PropTypes.bool,
   disableTooltip: PropTypes.bool,
   formDirty: PropTypes.bool,
+  connectDragSource: PropTypes.func.isRequired,
+  connectDragPreview: PropTypes.func.isRequired,
```

Comment:
Since these are required it constitutes a breaking change to this class. It's unlikely that anyone has extended this class in 4.0 since its release, but noting it anyway.

Reply:
Yeah that's a good point. It might be slightly different considering those two are provided with an HoC, but any extension will have to implement the DragSource HoC now anyway.